### DISCUSSION TITLE
Version 1.14.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#2290] [Changelog]
## Bugfixes
- #2148 editor-devtools is incompatible with editor-theme3. (prio-4)
- #2207 editor-theme3 does not affect block dropdown borders. (prio-4)
- #2207 editor-theme3 is incompatible with editor-devtools (prio-4)
- #2214 forum-time-zones does not translate "yesterday" and "today". (prio-4)
- #2234 drag-and-drop description does not mention new features. (prio-3)
- #2241 editor-dark-mode affects stage monitors and ask prompts. (prio-3)
- #2241 editor-dark-mode settings names are confusing. (prio-3)
- #2245 editor-stage-left rounds the corners. (prio-4)
- #2259 editor-devtools help message has typos. (prio-4)
- #2259 editor-devtools still refers to griffpatch. (prio-4)
- #2259 editor-devtools used wrong message in help. (prio-4)
- #2260 Themes show "for editor" tags in "All" mode on settings page. (prio-4)
- #2260 Color pickers of disabled addons' settings can be opened. (prio-4)
- #2260 Reset button is shown on disabled addons' settings. (prio-4)
- #2260 Disabled buttons change style when hovered. (prio-4)
- #2268 Addons perform expensive operations on DOM changes. (prio-3)
- #2274 Popup is incorrectly sized in Firefox when opened from overflow menu. (prio-4)
- #2278 Settings are loaded one by one when using presets or default values. (prio-4)
- #2279 "View Changelog" in update notification is capitalized incorrectly in some languages. (prio-4)
- #2280 editor-devtools does not follow capitalization rules. (prio-4)
- #2309 custom-zoom miscalculates zoom. (prio-3)
- #2315 Color pickers in settings cannot be closed by clicking. (prio-3)
- #2323 Color pickers in settings cap brightness at 99%. (prio-4)
- #2326 exact-count's default setting pings external servers. (prio-3)
- #2340 Disabling an addon without userscripts causes other addons to appear as disabled. (prio-3)
- #2341 Enabling an addon causes the addon to appear as running on all pages. (prio-3)

## New Features
- #2081 New addon: material-forum-editor-buttons (prio-4)
- #2149 New addon: gamepad (prio-3)
- #2163 my-ocular: rebrand addon (prio-3)
- #2207 editor-theme3: add dark presets (prio-3)
- #2212 Prevent users from commenting "scratch addons". (prio-3)
- #2251 Show cat-blocks as normal addons. (prio-3)
- #2254 New addon: editor-sounds (prio-4)
- #2260 Add addon groups and subcategories. (prio-2)
- #2282 Remove a11y addon. (prio-3)
- #2286 New addon: remove-curved-stage-border (prio-4)
- #2326 Add more tags to addons. (prio-3)

**Comment by CST1229** [comment]
What is the hour time it will come out?

**Reply by cobaltt7** [reply]
Who knows? Some chrome users might not even get it until Wednesday.

**Reply by WorldLanguages** [reply]
Realistically, tomorrow for most users
